### ISSUE TITLE
Time consuming in lift step

### ISSUE THREAD

**Issue by pnziy** [weijielyu/Gaga#15]
Very good job!

I have been working on your method for a while and I got some questions.

May I know if taking more than 33 hours in ' lift ' step is normal?

My GPU is NVIDIA RTX 4500 Ada Generation
And my original data set is 6 pictures.

This is the output:

Training progress:   0%|▏                             | 40/10000 [08:10<34:04:36, 12.32s/it, Loss=0.9952864]

And I'd like to know how you handle the size of the original data.  Are all the pictures the same size as their mask? 

Looking forward to your help, thank you!

**Comment by weijielyu** [comment]
Hi, thanks for your interest. 33 hours is definitely not normal, lifting for 10000 steps should take about the same time as training GS for 10000 steps. You can check if your CPU is occupied and taking long time to load data.

For the image size, yes it should be the same size as the mask.

**Comment by pnziy** [comment]
Thanks for your reply! I changed the iterations to 10 and I got the result faster...

**Comment by weijielyu** [comment]
Cool, but I don't think the training will converge with only 10 iterations. You may want to check for the reason, i.e. loading images is slow or training is slow or saving images is slow.

**Comment by pnziy** [comment]
Yes... Actually, in the lift.py I uncomment your code, so I run with:

 #loss_obj = cls_criterion(logits.unsqueeze(0), gt_obj.unsqueeze(0)).squeeze().mean()
    
        try:
            loss_obj = cls_criterion(logits.unsqueeze(0), gt_obj.unsqueeze(0)).squeeze().mean()
        except:
            _, h, w = logits.shape
            gt_obj = torch.nn.functional.interpolate(gt_obj.unsqueeze(0).unsqueeze(0).float(), size=(h, w), mode="nearest").long().squeeze(0).squeeze(0)
            loss_obj = cls_criterion(logits.unsqueeze(0), gt_obj.unsqueeze(0)).squeeze().mean()

Could that be one of the reasons?
